delfin/drivers/api.py

```diff
@@ -95,15 +98,42 @@ def remove_trap_config(self, context, storage_id, trap_config):
 
     def parse_alert(self, context, storage_id, alert):
         """Parse alert data got from snmp trap server."""
-        driver = self.driver_manager.get_driver(context, storage_id=storage_id)
+        driver = self.driver_manager.get_driver(context,
+                                                storage_id=storage_id)
         return driver.parse_alert(context, alert)
 
     def clear_alert(self, context, storage_id, sequence_number):
         """Clear alert from storage system."""
-        driver = self.driver_manager.get_driver(context, storage_id=storage_id)
+        driver = self.driver_manager.get_driver(context,
+                                                storage_id=storage_id)
         return driver.clear_alert(context, sequence_number)
 
     def list_alerts(self, context, storage_id, query_para=None):
         """List alert from storage system."""
-        driver = self.driver_manager.get_driver(context, storage_id=storage_id)
+        driver = self.driver_manager.get_driver(context,
+                                                storage_id=storage_id)
         return driver.list_alerts(context, query_para)
+
+    def collect_array_metrics(self, context, storage_id, interval,
+                              is_history):
+        """Collect array performance metrics"""
+        driver = self.driver_manager.get_driver(context,
+                                                storage_id=storage_id)
+        return driver.collect_array_metrics(context, storage_id,
+                                            interval, is_history)
+
+    def collect_pool_metrics(self, context, storage_id, interval,
```

Comment:
we can remove this interface for now, let us do only array leval interface in this PR